src/usr/local/www/vpn_ipsec_phase1.php

```diff
@@ -171,8 +171,9 @@
 if (!is_array($pconfig['encryption']['item']) || count($pconfig['encryption']['item']) == 0) {
 	$item = array();
 	$item['encryption-algorithm'] = array('name' => "aes");
-	$item['hash-algorithm'] = "sha1";
-	$item['dhgroup'] = "2";
+	$item['encryption-algorithm'] = array('keylen' => 128);
```

Comment:
One more little nit upon further review, this should still specify `'name' => "aes"` in addition to the `keylen`. The browser may appear to do the correct thing here, but it shouldn't rely on the default selection behavior of the browser there.

Reply:
That was actually my intent, but I somehow missed that I was overwriting the first set of `encryption-algorithm` with the second operation.